### DISCUSSION TITLE
OpenCorePkg upgrade

### DISCUSSION THREAD

**Discussion by serdeliuk** [CloverHackyColor/CloverBootloader#326] [Ideas]
Hello,
I started to try to upgrade to latest OpenCorePkg is anyone else working on this?
I wonder if this is not already done and in pending to be released.

I already saw different kind of issues as for example lodepng library has 3 different versions, one in OC, one in rEFIt and other one in theme manager....

Best regards,

Marc

**Comment by chris1111** [comment]
nice test keep us informed of the tests

**Comment by serdeliuk** [comment]
I was able to upgrade to OpenCore 0.6.4 with Clover 5127, it compile and boot well.
I am preparing a new fork/test-branch and later a pull request, let hope that i will have time tomorrow :)

**Comment by serdeliuk** [comment]
I have prepared all required stuff and requested a pull, meanwhile anyone interested in modifications can check my forks

obsolete://github.com/serdeliuk/OpenCorePkg/tree/0.6.4-Clover
obsolete://github.com/serdeliuk/CloverBootloader/tree/OC-0.6.4

**Comment by serdeliuk** [comment]
O.C-0.6.5 was released so I upgraded as well
https://github.com/serdeliuk/OpenCorePkg/tree/0.6.5-Clover
 https://github.com/serdeliuk/CloverBootloader/tree/OC-0.6.5

**Comment by serdeliuk** [comment]
I re-upgraded the OpenCore to the latest Clover. anyone who want that is here now. Clover 5139 (master) with OC 0.6.5 that allow to boot  any OS with SecureBootModel enabled. (you need to enable misc in main.cpp then create the misc section in config-oc.plist)
- https://github.com/serdeliuk/CloverBootloader
- https://github.com/serdeliuk/OpenCorePkg

**Comment by chris1111** [comment]
Why not 0.7.3 ?

**Reply by serdeliuk** [reply]
> Why not 0.7.3 ?

Since 0.6.6 they changed a lot, moved files, renamed a lot, and even removed others, 0.6.5 was the max version upgrade with minimum effort. I will try to progressively upgragrade, but will be harder with each version and i do not have required time.

**Comment by serdeliuk** [comment]
Good, twelve is a good number...
